src/main/resources/application.yaml

```diff
@@ -181,9 +181,6 @@ scheduling:
       enabled: ${DELETE_COMPLETE_FILES_ENABLED}
       cron: ${DELETE_COMPLETE_FILES_CRON}
     reupload:
-      batch: ${REUPLOAD_BATCH:20} # fetch 20 envelopes at once to process them. 0 for all
-      delay: ${REUPLOAD_DELAY:1800000} # In milliseconds, 30min
-      enabled: ${REUPLOAD_ENABLED:false}
       max_tries: ${REUPLOAD_MAX_TRIES:5}
```

Comment:
i guess it will be merged to `upload` section as there is no `reupload` task anymore

Reply:
yup, I guess my comment on `main.tf` should be here instead